.github/workflows/wasm-sdk-build.yml

```
@@ -81,9 +81,23 @@ jobs:
       - name: Install wasm-opt
         run: |
           if ! command -v wasm-opt &> /dev/null; then
-            echo "Installing wasm-opt..."
-            sudo apt-get update
-            sudo apt-get install -y binaryen
+            echo "Installing wasm-opt from GitHub releases..."
+            # Get the latest release version
+            WASM_OPT_VERSION=$(curl -s https://api.github.com/repos/WebAssembly/binaryen/releases/latest | grep -oP '"tag_name": "\K[^"]+')
+            echo "Installing wasm-opt version: $WASM_OPT_VERSION"
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)